inspirehep/config.py

```diff
@@ -443,6 +455,10 @@
 
 SEARCH_ELASTIC_DEFAULT_INDEX = 'hep'
 
+INSPIRE_PATH = pkg_resources.resource_filename("inspirehep", "")
```

Comment:
No I just thought that maybe someone will need this one day so I left it as extra information. You think we should remove it?